docs/migrating_to_version_3.md

```diff
@@ -18,6 +18,8 @@ Read on for the list of most interesting changes.
 
 - Changed the log format to JSON :sunglasses:
     - The logging format of Connaisseur is now fully transformed into JSON. It includes a `timestamp` and a `message` field, next to additional fields depending on the context. Logs of the HTTP request do not contain a `message` field.
+- Different behavior on reading feature flags
```

Comment:
Would put to both major changes and API changes and add migration script
Would formulate more from an end user perspective, instead of implementation details. "Before any truthy value ..." or the like